scripts/acceptance-tests-ide.sh

```diff
@@ -2,46 +2,84 @@
 
 set -euo pipefail
 
-readonly TOOLSMITH_ENV_INPUT="${TOOLSMITH_ENV_INPUT:?must be set}"
-readonly DEFAULT_ORG="${DEFAULT_ORG:?must be set}"
-readonly DEFAULT_SPACE="${DEFAULT_SPACE:?must be set}"
-readonly SKIP_SSL_VALIDATION="${SKIP_SSL_VALIDATION:?must be set}"
-
-declare API_HOST
-readonly API_PORT=443
-readonly USERNAME="admin"
-declare PASSWORD
-readonly CLIENT_ID="admin"
+readonly METADATA_FILE="${METADATA_FILE:?must be set}"
+
+declare CF_API_HOST
+declare CF_USERNAME="admin"
```

Comment:
Can these vars with fixed values be made `readonly`?

Reply:
Ah yes, thanks!